Makefile.core.mk

```diff
@@ -703,6 +703,20 @@ show.goenv: ; $(info $(H) go environment...)
 show.%: ; $(info $* $(H) $($*))
 	$(Q) true
 
+#-----------------------------------------------------------------------------
+# Target: custom resource definitions
+#-----------------------------------------------------------------------------
+
+API_TMP := $(shell mktemp -d -u)/api
+API_UPDATE_BRANCH ?= "master"
+
+update-crds:
+	@mkdir -p $(API_TMP)
+	@git clone -q --depth 1 --single-branch --branch $(API_UPDATE_BRANCH) https://github.com/istio/api $(API_TMP)
+	@rm -f crd-all.gen.yaml
+	@cp $(API_TMP)/kubernetes/customresourcedefinitions.gen.yaml $(shell pwd)/install/kubernetes/helm/istio-init/files/crd-all.gen.yaml
```

Comment:
I think we use another method to get the root in other places other than `pwd`,we may want to use that for consistency. I may be wrong though, I think this will probably work as well

Reply:
Changed to using `$REPO_ROOT`